enterprise-registry/github-app/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: docs
+title: Register GitHub Application
+category: registry
+sub_category: setup
+forkurl: https://github.com/coreos/docs/blob/master/enterprise-registry/github-app/index.md
+weight: 5
+---
+
+# Creating an OAuth Application in GitHub
+
```

Comment:
This doc should only ever be linked in from another doc, so I'm unsure what an overview will add